pages/controls/Changing_Registered_Email_Address_For_An_Account.md

```diff
@@ -0,0 +1,158 @@
+---
+
+layout: col-sidebar
+title: "Changing A User's Registered Email Address"
+author: "Philip H. Schlesinger"
+tags: ["control", "email", "address", "registered", "change", "account", "takeover"]
+
+---
+
+{% include writers.html %}
+
+People's email addresses regularly change.  The process below is the recommended method to implement in a system to handle that situation.  
+
+NOTE: The process is less stringent when using [Multifactor Authentication](https://cheatsheetseries.owasp.org/cheatsheets/Multifactor_Authentication_Cheat_Sheet.html) because the proof-of-identity is stronger than just using a password.
+
+## Process (Depending on a System's Level of Authentication)
+
+### Recommended Process If The User HAS [Multifactor Authentication](https://cheatsheetseries.owasp.org/cheatsheets/Multifactor_Authentication_Cheat_Sheet.html) Enabled
```

Comment:
I'd suggest not putting links in headings.

Reply:
Done

Reply:
If you feel the link is important then just add a lead in sentence before the process

Reply:
Nah, it's ok.  I link to it later on in the paragraphs